test_window_border_plain[hlwm_process0-xvfb0] endianess issue?

Note: this might be an endianess thing
```
>       assert img.pixel(0, 0) == color
E       assert (188, 159, 255) == (159, 188, 18)
E         At index 0 diff: 188 != 159
E         Use -v to get the full diff
```

Full log at
https://buildd.debian.org/status/fetch.php?pkg=herbstluftwm&arch=powerpc&ver=0.9.4-5&stamp=1653644891&raw=0

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -826,7 +826,7 @@ class X11:
         return w, self.winid_str(w)
 
     def screenshot(self, win_handle) -> RawImage:
-        """screenshot of a windows content, not including its border"""
+        """screenshot of a window's content, not including its border"""
         geom = win_handle.get_geometry()
         attr = win_handle.get_attributes()
         # Xlib defines AllPlanes as: ((unsigned long)~0L)
@@ -860,7 +860,14 @@ class X11:
             assert raw.depth in [32, 24]
             # both for depth 32 and 24, the order is BGRA
             pixelsize = 4
-            (blue, green, red) = (0, 1, 2)
+            # actually, there should be an attribute raw.byte_order which
+            # tells the byte order in the present XImage,
+            # but this is unfortunately missing in python-xlib. so let's
+            # work around this using the system's settings:
+            if sys.byteorder == 'little':
+                (blue, green, red) = (0, 1, 2)
+            else:
+                (blue, green, red) = (3, 2, 1)
             size = geom.width * geom.height
             assert len(raw.data) == pixelsize * size
             rgbvals = [(raw.data[pixelsize * (y * geom.width + x) + red],

